contracts/contracts/TACoApplication.sol

```diff
@@ -148,7 +147,7 @@ contract TACoApplication is IApplication, ITACoChildToRoot, OwnableUpgradeable {
         uint64 endCommitment;
     }
 
-    uint256 public constant REWARD_PER_TOKEN_MULTIPLIER = 10 ** 4;
+    uint256 public constant REWARD_PER_TOKEN_MULTIPLIER = 10 ** 3;
```

Comment:
This was some recommendation from the audit right, so instead of 10,000 we are doing 1000. Is that presumably still in line with what the original concern was in the audit?

Reply:
they suggested `uint256` and for `uint256` it could be 10^9, for `uint160` smaller and I tried to squeeze max because I still doubt overflow will ever be possible